src/main/java/edu/upf/taln/dri/lib/model/util/ExtractorFactory.java

```diff
@@ -0,0 +1,23 @@
+package edu.upf.taln.dri.lib.model.util;
+
+import edu.upf.taln.dri.module.importer.SourceENUM;
+
+public class ExtractorFactory {
+
```

Comment:
The Factory method picks the appropriate extractor based on its given enum. (This enum is created in `ObjectGenerator` based on the `sourceDocFeature` String available there.)